substrate/frame/offences/benchmarking/src/mock.rs

```diff
@@ -39,29 +39,11 @@ type Balance = u64;
 
 #[derive_impl(frame_system::config_preludes::TestDefaultConfig)]
 impl frame_system::Config for Test {
-	type BaseCallFilter = frame_support::traits::Everything;
-	type BlockWeights = ();
-	type BlockLength = ();
-	type DbWeight = ();
-	type RuntimeOrigin = RuntimeOrigin;
 	type Nonce = Nonce;
-	type RuntimeCall = RuntimeCall;
-	type Hash = sp_core::H256;
-	type Hashing = ::sp_runtime::traits::BlakeTwo256;
 	type AccountId = AccountId;
 	type Lookup = IdentityLookup<Self::AccountId>;
```

Comment:
AccountId is actually set to u64 above in this pallet too and can be removed
```suggestion
```

Reply:
There are a few more like this which I've suggested but not explained, most are types defined at the top of the file to be the same as the `TestDefaultConfig`